config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```
@@ -1358,3 +1358,126 @@ presubmits:
           requests:
             cpu: 4
             memory: 6Gi
+  - name: pull-kubernetes-cos-cgroupv2-containerd-node-e2e-features
+    skip_branches:
+      - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-dashboards: sig-node-presubmits
+      testgrid-tab-name: pr-cos-cgroupv2-containerd-node-e2e-features
+    always_run: false
+    optional: true
+    cluster: k8s-infra-prow-build
+    max_concurrency: 12
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221024-d0c013ee2d-master
+        args:
+        - --root=/go/src
+        - --repo=k8s.io/kubernetes
```

```suggestion
          - "--job=$(JOB_NAME)"
          - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
```

^ presubmits explicitly need the REFS config, not sure about the job name tho.

---

Thanks for the catch @endocrimes, updated, ptal!